src/mocks/healthServices.mock.json

```diff
@@ -0,0 +1,182 @@
+        "content": "U.S. Department of Veterans Affairs"
+      }
+    },
+    {
+      "tag": "meta",
+      "attributes": {
+        "name": "twitter:card",
+        "content": "summary_large_image"
+      }
+    },
+    {
+      "tag": "meta",
+      "attributes": {
+        "name": "twitter:site",
+        "content": "@DeptVetAffairs"
+      }
+    },
+    {
+      "tag": "meta",
+      "attributes": {
+        "name": "twitter:title",
+        "content": "Nashville Vet Center - Military sexual trauma care | Veterans Affairs"
+      }
+    },
+    {
+      "tag": "meta",
+      "attributes": {
+        "name": "twitter:image",
+        "content": "https://www.va.gov/img/design/logo/va-og-image.png"
+      }
+    },
+    {
+      "tag": "meta",
+      "attributes": {
+        "name": "twitter:image:alt",
+        "content": "U.S. Department of Veterans Affairs"
+      }
+    }
+  ],
+  "path": {
+    "alias": "/nashville-vet-center/service/nashville-vet-center-military-sexual-trauma-care",
+    "pid": 40773,
+    "langcode": "en"
+  },
+  "content_translation_source": "und",
+  "content_translation_outdated": false,
+  "field_body": {
+    "value": "<p>Military sexual trauma can happen to both men and women. If you experienced sexual assault or harassment during military service - no matter when you served – we provide counseling and treatment.</p>\r\n",
+    "format": "rich_text_limited",
+    "processed": "<p>Military sexual trauma can happen to both men and women. If you experienced sexual assault or harassment during military service - no matter when you served – we provide counseling and treatment.</p>\n"
+  },
+  "field_enforce_unique_combo": false,
+  "field_last_saved_by_an_editor": null,
+  "links": {
+    "self": {
+      "href": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/node/vet_center_facility_health_servi/e03928d8-ec40-4f0d-b53a-515d4625aa2c?resourceVersion=id%3A518499"
+    }
+  },
+  "node_type": {
+    "type": "node_type--node_type",
+    "id": "c04196ba-18ef-4627-b057-966bda224bb1",
+    "resourceIdObjMeta": {
+      "drupal_internal__target_id": "vet_center_facility_health_servi"
+    }
+  },
+  "revision_uid": {
+    "type": "user--user",
+    "id": "612f6a71-2e48-4d71-884b-58ec5b533e6d",
+    "resourceIdObjMeta": {
+      "drupal_internal__target_id": 2019
+    }
+  },
+  "uid": {
+    "type": "user--user",
+    "id": "612f6a71-2e48-4d71-884b-58ec5b533e6d",
+    "resourceIdObjMeta": {
+      "drupal_internal__target_id": 2019
+    }
+  },
+  "field_administration": {
+    "type": "taxonomy_term--administration",
+    "id": "27a3b37c-2a81-4736-8027-5488611fbbd4",
+    "resourceIdObjMeta": {
+      "drupal_internal__target_id": 396
+    }
+  },
+  "field_office": {
+    "type": "node--vet_center",
+    "id": "e16bc5b7-dd30-481a-8dfd-0ff288e64024",
+    "resourceIdObjMeta": {
+      "drupal_internal__target_id": 3877
+    }
+  },
+  "field_service_name_and_descripti": {
```

Comment:
re-creating the mock with this field included may help simplify some of the field naming / typing elsewhere

Reply:
Added